src/docker-compose.override.yml

```diff
@@ -0,0 +1,9 @@
+version: '3.4'
+
+services:
+  arcus.template.webapi:
+    environment:
+      - ASPNETCORE_ENVIRONMENT=Development
+      - ARCUS_HTTP_PORT=5000
```

Comment:
Huh, I probably missed that but where is this being used? You still need to add that, right?